extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -141,6 +141,19 @@ define('ext.wikia.adEngine.adContext', [
 			!context.targeting.wikiIsCorporate
 		);
 
+		// Override prefooters sizes
+		context.opts.overridePrefootersSizes =  !!(
```

Comment:
What do you think about [adding a unit tests for those two cases as well](https://github.com/Wikia/app/blob/ADEN-2841-extend-prefooter-left/extensions/wikia/AdEngine/js/spec/AdContext.spec.js)?